src/ucs/datastruct/ptr_map.inl

```diff
@@ -53,6 +53,18 @@ static inline void ucs_ptr_map_destroy(ucs_ptr_map_t *map)
     kh_destroy_inplace(ucs_ptr_map_impl, &map->hash);
 }
 
+/**
+ * Returns whether the key is indirect or not.
+ *
+ * @param [in]  key     Key to object pointer.
+ *
+ * @return 0 - direct, otherside - indirect.
+ */
+static inline int ucs_ptr_map_key_indirect(ucs_ptr_map_key_t key)
```

Comment:
UCS_F_ALWAYS_INLINE

Reply:
other functions have `inline`
is it ok to declare this function with `UCS_F_ALWAYS_INLINE`?

Reply:
never mind, I was wrong
fixed